packages/cli/src/controllers/owner.controller.ts

```diff
@@ -125,10 +125,10 @@ export class OwnerController {
 		return sanitizeUser(owner);
 	}
 
-	@Post('/dismiss-v1')
-	async dismissBanner() {
-		await this.settingsRepository.saveSetting('ui.banners.v1.dismissed', JSON.stringify(true));
-
-		return { success: true };
+	@Post('/dismiss-banner')
+	async dismissBanner(req: OwnerRequest.Post) {
```

Comment:
`OwnerRequest.Post` seems like the wrong payload type here.
Maybe we should add a new `OwnerRequest.DismissBanner` type.

```typescript
    type BannerName = 'V1' | 'AnotherBanner';
    type Post = AuthenticatedRequest<
		{},
		{},
		Partial<{
			bannerName: BannerName
		}>,
		{}
	>;
```